modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Grid/Grid.es.js

```diff
@@ -50,6 +50,8 @@ const TableBodyColumns = ({
 					checked={column.value === value[row.value]}
 					className="form-builder-grid-field"
 					data-name={row.value}
+					data-option-reference-column={column.reference}
+					data-option-reference-row={row.reference}
```

Comment:
The changes look good to me. We are just clarifying a doubt about the separation of the row and column properties in the Grid field, to understand if this covers the use cases considered by product.

More details [here](https://liferay.slack.com/archives/C026UR5976G/p1725990673721729)

Reply:
Question answered ✅